### DISCUSSION TITLE
Attachment can not be downloaded

### DISCUSSION THREAD

**Discussion by Melkin1968** [edumeet/edumeet#778] [Q&A]
Once attached to a file the download button appears. Once click on the button it shows an empty progress bar and the download is not continuing. Own tracker doesn`t help. The difference is only that in case I own a tracker I can download files that I share. But I can`t download files shared by other people. The same situation is on edumeet.geant.org deployment. We use edumeet 3.4.0 version. The link is edumeet.uran.ua

**Comment by so010** [comment]
Just testet on edumeet.uran.ua and it worked.
Usually the tracking server does not work properly, second reason is turn service is not working and behind restrictive firewall, but then audio/video is is not working as well. We try to integrate tracker-service directly into edumeet - will soon be fixed.

edumeet.geant.org uses default tracker which is not available anymore.

**Comment by Melkin1968** [comment]
Dear Stefan,

Thank you for your kind prompt reply. I will wait for a new version and always ready to deploy new software in our NREN.
Best regards,
Yevhenii

**Comment by hkizilhan** [comment]
You can use another torrent tracker to solve this.
wss://tracker.openwebtorrent.com, i suggest.

it must be set in both server and app config files.

**Comment by accidentul** [comment]
@Melkin1968 i have the same issue how did you fixed it ?
